docs/data/material/components/dialogs/DraggableDialog.tsx

```diff
@@ -9,12 +9,14 @@ import Paper, { PaperProps } from '@mui/material/Paper';
 import Draggable from 'react-draggable';
 
 function PaperComponent(props: PaperProps) {
+  const nodeRef = React.useRef<HTMLDivElement>(null);
   return (
     <Draggable
+      nodeRef={nodeRef as React.RefObject<HTMLDivElement>}
```

Comment:
nodeRef doesn't accept a `null` type. If `null` is removed from the nodeRef declaration, TypeScript throws an error in the Paper component. Therefore, type assertion was used as a workaround.

Reply:
FYI there's an open [issue](https://github.com/react-grid-layout/react-draggable/issues/768) and [PR](https://github.com/react-grid-layout/react-draggable/pull/769) in the react-draggable repo about this TS error.

Reply:
Got it, FYI react-draggable last published version is on last year. I'm not sure if we have to wait for them to merge the PR.

https://www.npmjs.com/package/react-draggable

Reply:
Yeah! My intention wasn't for us to wait, just to be aware of it.